solutions/swb-app/src/dataSetRoutes.ts

```diff
@@ -106,7 +103,7 @@ export function setUpDSRoutes(router: Router, dataSetService: DataSetPlugin): vo
 
   // Get dataset
   router.get(
-    '/datasets/:dataSetId',
+    '/projects/:projectsId/datasets/:dataSetId',
```

Comment:
Do we have validations to make sure the dataset is related to the project in the url?

Reply:
We'll be relying solely on dynamic authz to enforce that the two are associated.

Reply:
so, does that mean that if we send a request with an authorized project with a non associated dataset , it will still return unauthorized?

Reply:
Yes, dynamic authz will thrown an unauthorized error if someone tries to pass in the projectId that was not set in the IdentityPermission at the time the dataset was created.